src/kolibri_android/kolibri_utils.py

```diff
@@ -24,6 +24,7 @@
 # correctly:
 REQUIRED_PLUGINS = [
     "kolibri.plugins.app",
+    "kolibri_android.plugin",
```

Comment:
Doing the migration through this empty plugin is interesting. Having a platform specific plugin has potential.